linux_os/guide/system/auditing/auditd_configure_rules/audit_login_events/audit_rules_login_events_faillog/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: sle12,ubuntu2004,ubuntu2204
+prodtype: sle12,sle15,ubuntu2004,ubuntu2204
```

Comment:
Will you add also a SLE 15 CCE?

Reply:
There is no CCE... yet. Maybe someone from the SUSE Maintainer Team can help us there.